### DISCUSSION TITLE
Does isar support a list of an embedded object

### DISCUSSION THREAD

**Discussion by IttaiBarkai** [isar/isar#942] [Help]
I created a collection where one property is a List of an embedded object I created. I have no issues storing the list of values in the isar database (which I confirmed by looking at the isar inspector), but I get a null error when I try and query the data. Maybe this is not a bug and simply not possible at the moment, but because I was able to store the data without any issues I thought maybe I should ask.

**Comment by abdullah-cse** [comment]
Please give some specific information. e.g the code and error message you get.
